gatorgrade/main.py

```diff
@@ -16,6 +19,8 @@ def gatorgrade(
     """Run the GatorGrader checks in the gatorgrade.yml file."""
     # check if ctx.subcommand is none
     if ctx.invoked_subcommand is None:
+        checks = parse_config(FILE)
+        checks = run_and_display_command_checks(commands)
```

Comment:
This is still not the correct argument for `run_and_display_command_checks`. The argument for `run_and_display_command_checks` should be the value returned by `parse_config`, which you store in the variable `checks`.